app/src/main/java/com/github/se/cyrcle/ui/report/ViewReportsScreen.kt

```diff
@@ -0,0 +1,167 @@
+package com.github.se.cyrcle.ui.report
+
+import android.annotation.SuppressLint
+import android.widget.Toast
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.FabPosition
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.model.parking.ParkingViewModel
+import com.github.se.cyrcle.model.report.Report
+import com.github.se.cyrcle.model.report.ReportedObjectType
+import com.github.se.cyrcle.model.report.ReportedObjectViewModel
+import com.github.se.cyrcle.model.review.ReviewViewModel
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import com.github.se.cyrcle.ui.navigation.Screen
+import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
+
+@SuppressLint("StateFlowValueCalledInComposition")
+@Composable
+fun ViewReportsScreen(
+    navigationActions: NavigationActions,
+    reportedObjectViewModel: ReportedObjectViewModel,
+    parkingViewModel: ParkingViewModel,
+    reviewViewModel: ReviewViewModel
+) {
+  val currentObject = reportedObjectViewModel.selectedObject.value
```

Comment:
You might want to use a `by reportedObjectViewModel.selectedObject.collectAsState()` here. The linter is here to help you and suppressing it for an error like this is not really recommended

Reply:
I'll give it a try